chart/compass/templates/additional-prometheus-rules.yaml

```diff
@@ -149,4 +149,4 @@ spec:
           expr: delta(compass_ordaggregator_compass_ord_aggregator_job_sync_failure_number[1m]) >= 0
           for: 0m
           labels:
-            severity: warning
\ No newline at end of file
+            severity: error
```

Comment:
what is that used for?

Reply:
This is the severity for the slack notifications coming from the ORD aggregation. Previously it was warning because the errors consisted of validation errors, but now the will be only runtime errors so that this changed to `error`.